### PR TITLE
Create discord-release-msg.yml

### PR DESCRIPTION
**Pull request by polymo1**
Sends message to Discord server by webhook.

### Changes

- added discord-release-msg

### Todo before merge

(or after): add discord webhook stuff

### CODE REVIEW

**Comment by WesleyBryie** [comment]
> Builds for this Pull Request are available at [SideStore-0.5.0-pr.436+17f1f64.ipa](https://github.com/SideStore/SideStore/suites/14853465557/artifacts/845825111) [SideStore-0.5.0-pr.436+17f1f64-dSYM](https://github.com/SideStore/SideStore/suites/14853465557/artifacts/845825112) Have a nice day.

ah yes, _**fresh ipa.**_

**Comment by naturecodevoid** [comment]
We'll want to fork the action to create forum posts, which probably isn't possible with a webhook

**Comment by WesleyBryie** [comment]
Would like to point out that this all also applies to:

https://github.com/SideStore/SideServer-for-Linux/pull/7
and
https://github.com/SideStore/SideServer-Windows/pull/18

**Review by JoeMatt — APPROVED** [review]

File: .github/workflows/discord-release-msg.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+name: Release messages to discord announcement channel
+
+on: 
+  release:
+    types:
+      - created
+
+jobs:
+  run_main:
+    runs-on: ubuntu-18.04
```
Comment (SternXD) [inline .github/workflows/discord-release-msg.yml:11]: Is there a reason you chose Ubuntu 18.04?

**Review by SternXD — CHANGES_REQUESTED** [review]
I would like this addressed before I give my approval.

**Comment by WesleyBryie** [comment]
If needed for referenced, the action code is here: [https://github.com/nhevia/discord-styled-releases.](https://github.com/nhevia/discord-styled-releases)

**Comment by WesleyBryie** [comment]
https://github.com/discord/discord-api-docs/discussions/5610 This is relevant. If we make/find a bot or whatever that could do this automatically, that would make things greatly easier with the current setup.

**Review by JoeMatt — APPROVED** [review]

**Comment by WesleyBryie** [comment]
Just wanna point out, the current discord setup isn't compatible with this workflow. Further discussion is needed, we need to decide if we want to change the server or make a bot/webhook.

**Comment by Spidy123222** [comment]
> Just wanna point out, the current discord setup isn't compatible with this workflow. Further discussion is needed, we need to decide if we want to change the server or make a bot/webhook.

Should we keep this open?

**Comment by polymo1** [comment]
> Should we keep this open?

Honestly, I don't think it needs to stay open. We haven't discussed it any, and it would require something probably with SideBot, which doesn't need an action change in the repo.

Closing for not planned.

**Pull request closed without merging**